### PR TITLE
add Twerion.net

### PR DESCRIPTION
**Pull request by totKing**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [x] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/twerion/background.png
Comment (DoJapHD) [inline minecraft_servers/twerion/background.png:1]: Compress file

File: minecraft_servers/twerion/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/twerion/background@2x.png:1]: Compress file

File: minecraft_servers/twerion/manifest.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    ],
+    "supported_languages": [
+       "de",
+       "en",
+	   "ru"
+    ],
+    "social": {
+      "web": "https://twerion.net",
+      "web_shop": "https://shop.twerion.net",
+	  "web_support": "https://forum.twerion.net",
```
Comment (DoJapHD) [inline minecraft_servers/twerion/manifest.json:20]: Web Support can't be forum; add FAQ or contact form

File: minecraft_servers/twerion/manifest.json
Comment (DoJapHD) [inline minecraft_servers/twerion/manifest.json:1]: Fix indentations

File: minecraft_servers/twerion/manifest.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	  "%.twerion.eu",
+	  "%.twerion.org"
+    ],
+    "supported_languages": [
+       "de",
+       "en",
+	   "ru"
+    ],
+    "social": {
+      "web": "https://twerion.net",
```
Comment (DoJapHD) [inline minecraft_servers/twerion/manifest.json:18]: Can't reach website

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/twerion/manifest.json (hunk 2 above)
Comment (totKing) [reply minecraft_servers/twerion/manifest.json:18]: Corrected.

File: minecraft_servers/twerion/manifest.json
Comment (totKing) [reply minecraft_servers/twerion/manifest.json:1]: It was checked online again by a json validator. Should fit.

File: minecraft_servers/twerion/manifest.json (hunk 1 above)
Comment (totKing) [reply minecraft_servers/twerion/manifest.json]: Removed.

File: minecraft_servers/twerion/background@2x.png
Comment (totKing) [reply minecraft_servers/twerion/background@2x.png:1]: Done.

File: minecraft_servers/twerion/background.png
Comment (totKing) [reply minecraft_servers/twerion/background.png:1]: Done.

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**